src/Umbraco.Web.UI.Client/src/views/components/tabs/umb-tabs-nav.html

```diff
@@ -1,13 +1,16 @@
 <ul role="tablist" class="umb-tabs-nav">
-    <li ng-click="vm.clickTab($event, tab)" class="umb-tab" role="tab" aria-selected="true" tabindex="0" ng-repeat="tab in vm.tabs |  limitTo: vm.maxTabs" data-element="tab-{{tab.alias}}" ng-class="{'umb-tab--active': tab.active, 'umb-tab--error': tabHasError}" val-tab>
-        <a>{{ tab.label }}</a>
+    <li class="umb-tab" ng-repeat="tab in vm.tabs |  limitTo: vm.maxTabs" data-element="tab-{{tab.alias}}" ng-class="{'umb-tab--active': tab.active, 'umb-tab--error': tabHasError}" val-tab>
+        <button class="btn-reset umb-tab-button" ng-click="vm.clickTab($event, tab)" role="tab" aria-selected="{tab.active}">{{ tab.label }}</button>
     </li>
 
-    <li data-element="tab-expand" class="umb-tab umb-tab--expand" tabindex="0" ng-click="vm.toggleTray()" ng-class="{ 'open': vm.showTray }" ng-show="vm.needTray">
-        <a ng-href=""><i></i><i></i><i></i></a>
+    <li data-element="tab-expand" class="umb-tab umb-tab--expand" ng-class="{ 'open': vm.showTray }" ng-show="vm.needTray">
+        <button class="btn-reset umb-tab-button umb-tab-button--expand" ng-click="vm.toggleTray()">
+            <i></i><i></i><i></i>
+            <span class="sr-only"><localize key="visuallyHiddenTexts_tabExpand">View more options</localize></span>
+        </button>
         <umb-dropdown class="umb-tabs-tray" ng-if="vm.showTray" on-close="vm.hideTray()">
-            <umb-dropdown-item ng-repeat="tab in vm.tabs | limitTo: vm.overflowingTabs" ng-class="{'umb-tabs-tray-item--active': tab.active}" tabindex="0" ng-click="vm.clickTab($event, tab)">
-                <a ng-href="">{{ tab.label }}</a>
+            <umb-dropdown-item ng-repeat="tab in vm.tabs | limitTo: vm.overflowingTabs" ng-class="{'umb-tabs-tray-item--active': tab.active}">
+                <button class="btn-reset umb-tab-button" ng-click="vm.clickTab($event, tab)" role="tab" aria-selected="{tab.active}">{{ tab.label }}</button>
```

Comment:
Same for this button! :)

```suggestion
                <button type="button" class="btn-reset umb-tab-button" ng-click="vm.clickTab($event, tab)" role="tab" aria-selected="{tab.active}">{{ tab.label }}</button>
```